fbfmaproom/fbf-update-data.py

```diff
@@ -394,7 +394,20 @@
         # "a long time."
         'guatemala/vhi-aso',
         'https://iridl.ldeo.columbia.edu/home/.aaron/.nesdis/.VHP_16km/.VH/.VHI/X/-92.5/-88/RANGE/Y/13/18/RANGE/T/(1%20Jan%201996)/last/RANGE/T/(Aug-Oct)/seasonalAverage/',
-    ), 
+    ),
+   (
+        'guatemala/rain-may',
+        'http://iridl.ldeo.columbia.edu/SOURCES/.UCSB/.CHIRPS/.v2p0/.monthly/.global/.precipitation/X/-92.5/.1/-88/GRID/Y/13/.1/18/GRID/T/(May)/seasonalAverage//name//prcp_est/def/',
+    ),
+    (
+        # Static (non-updating) snapshot of
+        # http://iridl.ldeo.columbia.edu/SOURCES/.NOAA/.NESDIS/.STAR/.VHP/.16km/.VHI
+        # which is doubly broken: the upstram URL has changed, plus they had a
```

Comment:
upstram --> upstream, here and wherever you copied it from.

Reply:
Hi @aaron-kaplan , do you know what is left or missing to approve this pull request?

Reply:
It looks like @jturmelle reviewed this while I was away, so I'll pass the question to him.

I will point out, however, that you're replying to a comment where I pointed out a typo, and the typos still hasn't been fixed.

Reply:
My comment was the files they were pointing to didn't exist, so they couldn't have tested it yet in development mode.  I was waiting for them to state the testing was OK before approving and merging.